rest/tests/test_rest.py

```diff
@@ -0,0 +1,471 @@
+"""
+Unit tests for REST api
+"""
+
+from __future__ import unicode_literals
+
+import json
+from django.db.models.deletion import ProtectedError
+from django.test import Client
+
+from learningresources.tests.base import LoreTestCase
+from taxonomy.api import create_vocabulary
+from taxonomy.models import Vocabulary
+from rest.serializers import VocabularySerializer
+
+HTTP_OK = 200
+HTTP_CREATED = 201
+HTTP_NO_CONTENT = 204
+HTTP_BAD = 400
+HTTP_UNAUTHORIZED = 403
+HTTP_NOT_FOUND = 404
+HTTP_METHOD_NOT_ALLOWED = 405
```

Comment:
I know I did the same in my tests, but we should probably use [httplib](https://docs.python.org/2/library/httplib.html#httplib.HTTPS_PORT) for this